### DISCUSSION TITLE
[Bug]: cluster-ip as external listener not worked

### DISCUSSION THREAD

**Discussion by TigranKhudav** [strimzi/strimzi-kafka-operator#10449] [General]
### Bug Description

When I set nodeport it works fine but cluster-ip not worked 
I connected trough istio 
this is my log when connecting via node js setuped cluster-ip

`{"level":"ERROR","timestamp":"2024-08-14T08:26:55.456Z","logger":"kafkajs","message":"[Connection] Connection error: getaddrinfo ENOTFOUND kafka-kafka-node-pool-clusterip-2.kafka.svc","broker":"kafka-kafka-node-pool-clusterip-2.kafka.svc:9096","clientId":"my-app","stack":"Error: getaddrinfo ENOTFOUND kafka-kafka-node-pool-clusterip-2.kafka.svc\n    at GetAddrInfoReqWrap.onlookupall [as oncomplete] (node:dns:120:26)"}`

### Strimzi version

0.42.0

### Kubernetes version

1.29.1

### Installation method

helm chart

### Infrastructure

bare-metal

**Comment by scholzj** [comment]
Strimzi does not support Istio. Also, the error you are getting does not seem to have anything to do with Strimzi.
